libs/openFrameworks/utils/ofSystemUtils.cpp

```diff
@@ -753,131 +730,6 @@ string ofSystemTextBoxDialog(string question, string text){
 		 DestroyWindow(dialog);
 		 EnableWindow(WindowFromDC(wglGetCurrentDC()), TRUE);
 
-	#else
-
```

Comment:
why does all this code go away? i get that everything else is related to wide strings differences but this?